src/networkStats.ts

```diff
@@ -0,0 +1,66 @@
+const IMAGE_URL = `${process.env.REACT_APP_SOCKET_SERVER_URL}/test256.png`;
+const IMAGE_SIZE_BITS = 141978 * 8;
+const AVERAGE_MAX = 4;
+
+const speedHistory: number[] = [];
+
+const pushSpeed = (speed: number): void => {
+  speedHistory.push(speed);
+  if (speedHistory.length > AVERAGE_MAX) {
+    speedHistory.shift();
+  }
+};
+
+const getAverageSpeed = (): number => {
```

Comment:
This is calculating average so might be useful for other usecases, lets move it to util and rename it to `getAverage` ?

Reply:
and get's arrray as argument? sure.. move it..